server/src/test-data.js

```diff
@@ -11,7 +11,15 @@ export const loadTestData = () =>
         password: '123',
         email: 'test@test.com',
         organisation,
-      }),
+      }).then(user =>
+        Promise.all([
```

Comment:
this `.all()` is redundant but you probably know that

Reply:
Later down the road i will need further things in here, so this got it inline with the other test users